### ISSUE TITLE
Add tests for PR-465

### ISSUE THREAD

**Issue by DirkDevelopmentFreelance** [bccsa/luminary#571]
https://github.com/bccsa/luminary/pull/465

**Comment by DirkDevelopmentFreelance** [comment]
Placing in on hold for now as I have to come back to this, it was in "help needed" but forgot what help was needed, will revisit this ticket again on 20-01-2025

**Comment by DirkDevelopmentFreelance** [comment]
Add tests for:

- [ ] GroupSelector.vue: The list of assignable groups should only include groups to which the user has group assign access, and edit access to the docType in question (passed as prop)
- [ ] API validateChangeRequestAccess: Add tests for Language doctype validations
- [ ] API processChangerequest: Check if the API can reset the default language flag (see line 134) - if not implemented already

**Comment by DirkDevelopmentFreelance** [comment]
> API validateChangeRequestAccess: Add tests for Language doctype validations
Can't find code for this, should this still be implemented?

**Comment by oslabbie** [comment]
> > API validateChangeRequestAccess: Add tests for Language doctype validations
> > Can't find code for this, should this still be implemented?

Hey @DirkDevelopmentFreelance 

The code for this is in api > src > chaneRequests > validateChangeRequestAccess.spec.ts

I think this test is still needed

**Comment by DirkDevelopmentFreelance** [comment]
> > > API validateChangeRequestAccess: Add tests for Language doctype validations
> > > Can't find code for this, should this still be implemented?
> 
> Hey [@DirkDevelopmentFreelance](https://github.com/DirkDevelopmentFreelance)
> 
> The code for this is in api > src > chaneRequests > validateChangeRequestAccess.spec.ts
> 
> I think this test is still needed

Yes it seems I never comitted those changes. Will see if I can quickly get it in

**Comment by ivanslabbert** [comment]
Replaced by #807
